data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/LengthExpressionFunction.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import java.util.List;
+import javax.inject.Named;
+
+@Named
+public class LengthExpressionFunction implements ExpressionFunction {
```

Comment:
Let's be sure to add documentation for this new feature here: https://github.com/opensearch-project/data-prepper/blob/main/docs/expression_syntax.md

Reply:
Please also create a GitHub issue in this repo for us to update the syntax in the user-facing documents.

https://github.com/opensearch-project/documentation-website